macros/OD.mac

```diff
@@ -0,0 +1,132 @@
+# Sample setup macro with no visualization
```

Comment:
I think it might be better to keep this as OD.mac because this is the default baseline design. We are currently simulating an additional 10k geometry which is why there is also an OD_10k.mac. What do you think?